tests/js/ttimes.nim

```diff
@@ -36,8 +36,8 @@ let utcPlus2 = Timezone(zoneInfoFromUtc: staticZoneInfoFromUtc, zoneInfoFromTz:
 block timezoneTests:
   let dt = initDateTime(01, mJan, 2017, 12, 00, 00, utcPlus2)
   doAssert $dt == "2017-01-01T12:00:00+02:00"
-  doAssert $dt.utc == "2017-01-01T10:00:00+00:00"
```

Comment:
Why are there so fewer tests here?

Reply:
Do you mean why there are so few tests in `tests/js/ttimes.nim`? The times tests should probably just be merged into a single file that runs for both C & JS. I can fix it in a separate PR.